airbyte-integrations/connectors-performance/source-harness/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM airbyte/integration-base-java:dev AS build
+WORKDIR /airbyte
+ENV APPLICATION source-harness
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+RUN tar xf ${APPLICATION}.tar --strip-components=1 && rm -rf ${APPLICATION}.tar
+FROM airbyte/integration-base-java:dev
+WORKDIR /airbyte
+ENV APPLICATION source-harness
+COPY --from=build /airbyte /airbyte
+COPY base.sh .
+
+RUN curl -O https://s3.us-west-2.amazonaws.com/amazon-eks/1.25.6/2023-01-30/bin/linux/amd64/kubectl
```

Comment:
nit: comment why we need kubectl

Reply:
Done. 
Tweaked also build to curl the correct kubectl architecture.
for the arm64 enthusiasts among us